notebooks/NIRSPEC/BOTS/JWPipeNB-NIRSpec-BOTS.ipynb

```diff
@@ -2033,18 +2048,22 @@
    "source": [
     "# Read the white light curve.\n",
     "data = Table.read(stage3_whtlt[0], format=\"ascii.ecsv\")\n",
-    "xx = data[\"MJD_UTC_NRS1\"]\n",
-    "yy = data[\"whitelight_flux_NRS1\"] / np.median(data[\"whitelight_flux_NRS1\"][:100])\n",
+    "mjd = 0.5 * (data[\"MJD_UTC_NRS1\"] + data[\"MJD_UTC_NRS2\"])\n",
+    "\n",
+    "# Extract and normalize NRS1 and NRS2 white light fluxes.\n",
+    "flux_nrs1 = data[\"whitelight_flux_NRS1\"] / np.nanmedian(data[\"whitelight_flux_NRS1\"][:100])\n",
+    "flux_nrs2 = data[\"whitelight_flux_NRS2\"] / np.nanmedian(data[\"whitelight_flux_NRS2\"][:100])\n",
+    "combined_flux = 0.5 * (flux_nrs1 + flux_nrs2)\n",
```

Comment:
It looks like the NRS2 data is both noisier than the NRS1 and systematically offset after the tilt event.  Does it make sense to combine them like this?

Reply:
I think it could still be instructive to plot both, but an alternative might be to plot them separately on the same plot.  It might help to add a check to see if there are NRS2 white light flux columns in the data and only plot it if it is available so that this will run through on datasets that only have NRS1 data.

Reply:
Update; since I'm aiming to tag the notebooks shortly so work can begin on 12.1 notebooks I'll merge this as-is and then address the label issue separately.